drupal_test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -x
+set -e
+
+cd /tmp
+composer create-project --no-interaction drupal-composer/drupal-project:8.x-dev drupal
+cd drupal
+composer config repositories.pr path $HOME
```

Comment:
is `$HOME` the actual location of the cloned repo ? I don't think so (and this would not be reusable locally). I suggest reading the location with `pwd` before changing to `/tmp` instead.